solidity/.prettierrc.yaml

```diff
@@ -0,0 +1,2 @@
+semi: false
+trailingComma: "all"
```

Comment:
We shouldn't have a custom `.prettierrc`.

Reply:
Nice! I just copy-pasted it from tbtc 😉

Reply:
Yeah, it predates my adding prettier to eslint and I haven't migrated it yet :grimacing: